wgpu-types/src/lib.rs

```diff
@@ -207,6 +207,17 @@ pub const QUERY_SET_MAX_QUERIES: u32 = 4096;
 #[doc = link_to_wgpu_docs!(["query"]: "struct.QuerySet.html")]
 pub const QUERY_SIZE: u32 = 8;
 
+/// The minimum allowed value for [`AdapterInfo::subgroup_min_size`].
+///
+/// See <https://gpuweb.github.io/gpuweb/#gpuadapterinfo>
+/// where you can always use these values on all devices
+pub const MINIMUM_SUBGROUP_MIN_SIZE: u32 = 4;
+/// The maximum allowed value for [`AdapterInfo::subgroup_max_size`].
+///
+/// See <https://gpuweb.github.io/gpuweb/#gpuadapterinfo>
+/// where you can always use these values on all devices.
+pub const MAXIMUM_SUBGROUP_MAX_SIZE: u32 = 128;
```

Comment:
Q: Is it possible to use more than this on _some_ machines? If so, do we have any follow-up work for permitting larger groups?